tests/functional/api/groups/test_read.py

```diff
@@ -0,0 +1,114 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+import pytest
+import base64
+
+
+from h.models.auth_client import GrantType
+
+native_str = str
+
+
+@pytest.mark.functional
+class TestReadGroup(object):
```

Comment:
Given that the configuration and logic needed to make the API work is spread across several places, I do think having integration tests like this is necessary. However there is also a lot more setup happening per test case. Each of these tests takes a mean of ~0.5s to run on my system. The unit tests for the view take a mean of 0.1s per test in comparison.

I'm not asking for any changes here, just making a note for our collective awareness.